src/themes/money/theme.json

```diff
@@ -639,40 +639,60 @@
   },
 
   "styles": {
-    "h1": {
+    "heading": {
       "fontFamily": "heading",
-      "fontSize": "xxxl",
+      "color": "plum",
       "lineHeight": "heading",
       "fontWeight": "base",
-      "marginY": "md"
+      "marginTop": 0,
+      "marginBottom": 0
+    },
+
+    "h1": {
+      "variant": "styles.heading",
+      "fontSize": "xxxl",
+      "paddingTop": "xl",
+      "paddingBottom": "sm"
     },
 
     "h2": {
-      "variant": "styles.h1",
-      "fontSize": "xxl"
+      "variant": "styles.heading",
+      "fontSize": "xxl",
+      "paddingTop": "sm",
+      "paddingBottom": "sm"
     },
 
     "h3": {
-      "variant": "styles.h1",
-      "fontSize": "xl"
+      "variant": "styles.heading",
+      "fontSize": "xl",
+      "paddingTop": "sm",
+      "paddingBottom": "sm"
     },
 
     "h4": {
-      "variant": "styles.h1",
+      "variant": "styles.heading",
       "fontSize": "lg",
-      "marginY": "sm"
+      "marginY": "sm",
+      "paddingBottom": "sm"
```

Comment:
Is the mix of padding and margin intentional?

Reply:
No missed that - good spot